tests/core/ast_transforms/test_processor.py

```diff
@@ -0,0 +1,53 @@
+
+import ast
+from collections import Iterable
+
+from pytest import raises
+
+from fastats.core.ast_transforms.processor import recompile, uncompile
+
+
+def test_recompile_happy_path():
+    func_as_string = '''
+def f(n):
+     # This is a comment
+     return n ** 2
+'''
```

Comment:
The zero indent bothers me slightly. Do you think we could use [`textwrap.dedent`](https://docs.python.org/3.5/library/textwrap.html#textwrap.dedent) here and in the function below?

Reply:
No problem - implemented.